sentry-rails/lib/sentry/rails/controller_transaction.rb

```diff
@@ -2,10 +2,32 @@ module Sentry
   module Rails
     module ControllerTransaction
       def self.included(base)
-        base.prepend_before_action do |controller|
-          if Sentry.initialized?
-            Sentry.get_current_scope.set_transaction_name("#{controller.class}##{controller.action_name}", source: :view)
+        base.prepend_around_action(:sentry_around_action)
+      end
+
+      def sentry_around_action
+        if Sentry.initialized?
+          transaction_name = "#{self.class}##{action_name}"
+          Sentry.get_current_scope.set_transaction_name(transaction_name, source: :view)
+          Sentry.with_child_span(op: "view.process_action.action_controller", description: transaction_name) do |child_span|
+            if child_span
+              yield.tap do
+                child_span.set_http_status(response.status)
+                child_span.set_data(
+                  :payload,
+                  {
+                    format: request.format,
+                    method: request.method,
+                    path: request.path
+                  }
```

Comment:
were we also sending `params` before?
https://guides.rubyonrails.org/active_support_instrumentation.html#process-action-action-controller

Reply:
Added 👍